templates/pages/core/get-started/intel-joule.md

```diff
@@ -42,3 +44,25 @@ We will walk you through the steps of flashing Ubuntu Core on an Intel Joule. At
 * Remove all USB flash drives and reboot the system, it will reboot from the internal memory now containing Ubuntu Core
 
 {% include "includes/markdown/get_started_first_boot.md" %}
+
+## Alternative install: Ubuntu Desktop 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Desktop 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [Intel Joule - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/tuchuck/tuchuck-20161014085519.img.xz)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
+
+### Installation instructions
+
+Booting the board from the USB flash drive will start the Ubuntu installer.
+
+1. Boot the system from the USB flash drive
+* The system will automatically execute the first stage of installation, including eMMC storage partitioning and image installation. After installation is complete, a prompt dialog will be shown and you will need to restart the system
```

Comment:
No need, Markdown interprets it as a numbered list as long as it starts with a 1.